File: src/main/java/me/Geek/api/hook/hookItemsAdder.java
package me.Geek.api.hook;

import dev.lone.itemsadder.api.CustomStack;
import me.Geek.GeekMail;
import org.bukkit.Bukkit;
import org.bukkit.inventory.ItemStack;

/**
 * 作者: 老廖
 * 时间: 2022/8/11
 **/
public final class hookItemsAdder {

    public static ItemStack getItemsAdder(String id) {
        return CustomStack.getInstance(id).getItemStack();
    }
    public static void hook() {
        GeekMail.say("&7软依赖 &fItemsAdder &7已兼容.");
    }

}

File: src/main/kotlin/me/geek/mail/api/hook/hookItemsAdder.kt
package me.geek.mail.api.hook

import dev.lone.itemsadder.api.Events.ItemsAdderFirstLoadEvent
import me.geek.mail.GeekMail
import me.geek.mail.GeekMail.say
import me.geek.mail.common.menu.Menu
import taboolib.common.platform.event.SubscribeEvent

/**
 * 作者: 老廖
 * 时间: 2022/8/11
 *
 **/
object hookItemsAdder {

    @SubscribeEvent
    fun onHook(e: ItemsAdderFirstLoadEvent) {
        say("&7软依赖 &fItemsAdder &7已兼容.")
        Menu.loadMenu()
    }
}